src/alert.ts

```diff
@@ -14,9 +14,16 @@ import {
   FdsSize2,
   FdsTokenSize3,
   uiHelperTextClass,
-} from '@fintraffic-design/coreui-css'
-import './global-types'
-import { FdsIconType } from './fds-icon'
+} from '@fintraffic/fds-coreui-css'
+import { FdsIcon, FdsIconType } from './icon.js'
+
+// Declare used elements to avoid lit-plugin(no-unknown-tag-name) warning
+// These components still need to be registered separately (by the importing app).
+declare global {
+  interface HTMLElementTagNameMap {
+    'fds-icon': FdsIcon
+  }
+}
```

Comment:
Unfortunate we have to do this everywhere with the icon (and possibly other reused components) just to get rid of this warning 😕 Personally not sure if it is worth it to keep 'no-unknown-tag-name'-warning activated at all. Or should we rather move all the these type declarations to their respective component files instead of the definition file?

Reply:
I will merge this for now. We can try to improve it later.

Reply:
Alright that's a sound reason to keep these here for now atleast. Merge away 🙂